CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # NEXT RELEASE
 
 ### Enhancements
+* Query parser supports property names containing white space. White space characters must be excapes with a '\'.
```

Comment:
maybe also note the removal of IncludeDescriptor, and that the interface for parsing string queries has changed

Reply:
Done